mos-platform/common/include/fixed_point.h

```diff
@@ -0,0 +1,301 @@
+
+
+#ifndef _FIXED_POINT_H
+#define _FIXED_POINT_H
+
+// Copyright 2024 LLVM-MOS Project
```

Comment:
Nit; the license header should be at the very start of the file; it applies to everything, including the include guard ;)